### PR TITLE
Create ui-c8y-1020-3-0-device-protocol-object-mapping-component-angular-migration-6482-graft-release-cd.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-2884](https://cumulocity.atlassian.net/browse/DM-2884)
Original PR: [6659](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6659)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-3-0-device-protocol-object-mapping-component-angular-migration-6482-graft-release-cd.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device protocol object mapping component angular migration (#6482) [GRAFT][release/cd] (#6659)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-3-0-device-protocol-object-mapping-component-angular-migration-6482-graft-release-cd.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Migrate device protocol object mapping component to Angular
```

File: content/change-logs/device-management/ui-c8y-1020-3-0-device-protocol-object-mapping-component-angular-migration-6482-graft-release-cd.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2884
+version: 1020.3.0
+---
+Device protocol object mapping component angular migration (#6482) [GRAFT][release/cd] (#6659)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-3-0-device-protocol-object-mapping-component-angular-migration-6482-graft-release-cd.md:17]: The Claude AI suggested to use the following body:
```suggestion
As part of the ongoing modernization efforts, the device protocol object mapping component has been migrated from AngularJS to Angular. This component is responsible for mapping device protocol objects to the internal representation used by the platform. The migration to Angular brings improved performance, maintainability and aligns the component with the latest web technologies and standards used in the platform. Functionally, the component behaves the same way as before the migration and no impact is expected for end users or existing device protocol object mappings.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by JacekGonera** [comment]
Docs not needed - user won't see any difference and the code won't be used until device protocol details page migration is merged.

**Pull request closed without merging**